### DISCUSSION TITLE
Unable to mirror images from quay.io

### DISCUSSION THREAD

**Discussion by veerlapati** [quay/quay#1631] [Q&A]
I am currently working on Quay repo Image mirroring from Quay.io to my Quay registry.
 
I am  trying to mirror the images from following quay.io repo URL
https://quay.io/repository/openshift-release-dev/ocp-release?tab=tags
 
My requirement is to mirror all the images of Version 4.11.x  from [quay.io/repository/openshift-release-dev/ocp-release](https://quay.io/repository/openshift-release-dev/ocp-release) to my quay registry

<img width="1369" alt="Screen Shot 2022-11-21 at 6 06 46 PM" src="https://user-images.githubusercontent.com/40397101/203189570-67dd660c-546f-4d9b-9f3e-311f86ea94b0.png">

 
I have tried different tag patterns and all of them are failing with following error
 
4.11.16-*
4.11.[16]*
4.12.0-rc.1-*
4.11.*
 
 
Error message: time="2022-11-21T16:09:14Z" level=fatal msg="Error parsing image name \"docker://quay.io/openshift-release-dev/ocp-release:latest\": reading manifest latest in quay.io/openshift-release-dev/ocp-release: manifest unknown: manifest unknown"

Need help in custom tag patterns for image tags.

Any help is highly appreciated.

Thanks

**Comment by LiZhang19817** [comment]
You can take a try with this tag pattern, "4.11.16-multi, 4.11.*"

**Reply by veerlapati** [reply]
Thanks, I have tried the above pattern 4.11.*

However, its not mirroring all tag versions above 4.11 instead only mirroring set of tags.

4.11.0-0.nightly-multi-2022-06-24-013136
4.11.0-0.art-assembly-rc.7-multi-2022-08-03-162248

![image](https://user-images.githubusercontent.com/40397101/203381523-f43f09c8-411b-45be-b3d6-651b460c44cd.png)

![image](https://user-images.githubusercontent.com/40397101/203381601-73bf099c-7d24-46ca-b71a-c44e00014c14.png)

Tag pattern:

![image](https://user-images.githubusercontent.com/40397101/203381194-cc1b65bd-830d-4e79-b7c3-c5c29ecfaf27.png)
